src/Commands.pm

```diff
@@ -56,6 +56,7 @@ our %customCommands;
 sub initHandlers {
 	%handlers = (
 	a					=> \&cmdAttack,
+	achievegetreward	=> \&cmdAchieveGetReward,
```

Comment:
Can we please use a smaller command name for this? Maybe `achieve` or `ach` as the base command, then `achieve list`, `achieve reward ID`, etc.